integration-tests/tests/workspace-dashboard.js

```diff
@@ -31,7 +35,11 @@ const workspaceDashboardPage = (testPage, token, workspaceName) => {
       await Promise.all(_.map(async tab => {
         await (enabled ? testPage.waitForXPath(navChild(tab)) : assertNavChildNotFound(testPage, tab))
       }, expectedTabs))
-    }
+    },
+
+    assertAzureWarning: async () => {
+      await testPage.waitForXPath(`//*[@id="azureWarningBanner"]/*[contains(text(), "${azureWarning}")]`, { visible: true })
```

Comment:
If you can do this with role alert instead of introducing an ID, that would be better. For example, @role='alert'  (haven't tested that it works).

Reply:
we wanted to be very specific with the xpath in case we add the same warning elsewhere. maybe role='alert' is specific enough though? 🤷

Reply:
Huh, it's hard for me to imagine that we would want the exact same message to appear twice on the screen. The benefits to using the role is that it verifies that the correct aria role was added, and it avoids introducing an id (ids must be unique on the page, and we usually randomly generate them).

In order for a duplicate to be a problem, we would have the exact same message on the page twice, both with role of alert. I feel like that is extremely unlikely.